### PR TITLE
cleaning up bash commands to make copy/paste cleaner

### PR DESCRIPTION
**Pull request by bd-dean**
removed '$' from bash sections to enable the copy/paste to work without edit.

### CODE REVIEW

File: source/clear-linux/tutorials/azure.rst [hunk 1]
```diff
@@ -436,11 +436,11 @@ For this tutorial, we are using the |CL| Basic SKU for our VM.
 
    You are now logged into your new |CL| VM as [user], where [user] is your
    user name. To check which software bundles are included with
-   this image, run the :command:`swupd bundle-list` command:
+   this VM image, run the :command:`swupd bundle-list` command inside the VM:
```
Comment (tpleavitt) [inline source/clear-linux/tutorials/azure.rst]: need to add "sudo" in front of the swupd command.  The bask example has this so this is just for consistency

Comment (bd-dean) [reply source/clear-linux/tutorials/azure.rst]: added sudo :)

Comment (tpleavitt) [reply source/clear-linux/tutorials/azure.rst]: thanks!  LGTM then...ship it!

**Comment by mvincerx** [comment]
LGTM.

**Pull request merged**